src/features/pages/options/vanilla/_faq-schema.ts

```diff
@@ -0,0 +1,96 @@
+import { localize } from 'components/localization'
+
+type contentType = 'FAQPage' | 'Question' | 'Answer'
+
+type AcceptedAnswerType = {
+    '@type': contentType
+    text: string
+}
+
+type MainEntityType = {
+    '@type': contentType
+    name: string
+    acceptedAnswer: AcceptedAnswerType
+}
+
+type FaqSchemaType = {
+    '@context': string
+    '@type': contentType
+    mainEntity: MainEntityType[]
+}
```

Comment:
Other options pages also need this type. We should make this type reusable.

Reply:
done. please review